spec/support/shared_examples/data_page_shared_example.rb

```diff
@@ -0,0 +1,7 @@
+shared_examples 'a data page' do |tables|
+  it 'contains all requires tables' do
+    tables.each do |table_name|
+      expect page.has_table? table_name
```

Comment:
I was just copying the spec page mentioned in the original issue. I'll swap this around.